README.md

```diff
@@ -1 +1,19 @@
-# bouncerscript
+Bouncerscript README
+
+[![Build Status](https://travis-ci.org/mozilla-releng/bouncerscript.svg?branch=master)](https://travis-ci.org/mozilla-releng/bouncerscript)
+[![Coverage Status](https://coveralls.io/repos/github/mozilla-releng/bouncerscript/badge.svg?branch=master)](https://coveralls.io/github/mozilla-releng/bouncerscript?branch=master)
+
+
+## deploy a new version
+
+1. Once your PR is reviewed and passes the tests, have one of the admins review & merge it
+2. Bump to new version in `version.json` and commit with a "%VERSION%" message
```

Comment:
Nit: There is no `version.json`. `s/json/txt` ?

Reply:
Let's be consistent with `version.txt` then as I see most of *scripts are. I'll need to amend https://github.com/mozilla-releng/beetmoverscript/blob/master/version.json and https://github.com/mozilla-releng/balrogscript/blob/master/version.json at some point too.  Good catch again!